src/rpc.rs

```diff
@@ -517,13 +543,26 @@ impl Cacheable for GetAccountInfo {
         state.account_info_request_limit.as_ref()
     }
 
+    // for getAccountInfo requests, we don't need to subscribe in case if the owner program exists,
+    // and there's already an active subscription present for it
+    fn has_active_subscription(&self, state: &State) -> SubscriptionActive {
+        let owner = state.accounts.get(&self.pubkey).and_then(|data| {
```

Comment:
Why is there an extra map lookup for owner if we have this data already?

Reply:
Hmm, I cannot seem to find that data, we have cached response, yes, but it's serialiazed, may be i'm missing something here...

Reply:
I suppose we can return owner from get_from_cache

Reply:
Yes, we can, but it's not really easy to make use of this value
for example https://github.com/bobs4462/cacherpc/blob/sub-deduplication/src/rpc.rs#L363 here we can use that value and pass it to `has_active_subscription`, but https://github.com/bobs4462/cacherpc/blob/sub-deduplication/src/rpc.rs#L442 in this invocation we don't have access to that value. It's possible to to keep it around, but overall it's going to be pretty messy in my opinion.

Reply:
Owner was added at call site of the `has_active_subscription`, to eliminate extra cache lookup